src/snowflake/snowpark/dataframe.py

```diff
@@ -568,6 +569,7 @@ def collect(
         ...  # pragma: no cover
 
     @df_collect_api_telemetry
+    @open_telemetry(name="Dataframe.collect")
```

Comment:
Can `open_telemetry` grab the function name automatically?

Reply:
No